database/mongoDB/redis.js

```diff
@@ -0,0 +1,30 @@
+const { loadProject } = require('./db.js');
+
+var redisClient = require('redis').createClient;
+
+// const db = require('./db.js');
+var hits = 0;
+var total = 0;

```

Comment:
Use let and const here - and more-descriptive variable names.

Reply:
fixed